### PR TITLE
Fixing scope migration issues

### PR DESCRIPTION
**Pull request by CrowleyRajapakse**
## Purpose
Fixing https://github.com/wso2/product-apim/issues/8496

### CODE REVIEW

File: migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/client/MigrateFrom310.java [hunk 1]
```diff
@@ -25,8 +25,7 @@
 import org.wso2.carbon.apimgt.migration.APIMigrationException;
 import org.wso2.carbon.apimgt.migration.client.sp_migration.APIMStatMigrationException;
 import org.wso2.carbon.apimgt.migration.dao.APIMgtDAO;
-import org.wso2.carbon.apimgt.migration.dto.ResourceScopeInfoDTO;
-import org.wso2.carbon.apimgt.migration.dto.ResourceScopeMappingDTO;
+import org.wso2.carbon.apimgt.migration.dto.*;
```
Comment (msm1992) [inline migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/client/MigrateFrom310.java:28]: shall we remove * imports ?

File: migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/dao/APIMgtDAO.java [hunk 2]
```diff
@@ -22,9 +22,7 @@
 import org.apache.commons.logging.LogFactory;
 import org.wso2.carbon.apimgt.impl.utils.APIMgtDBUtil;
 import org.wso2.carbon.apimgt.migration.APIMigrationException;
-import org.wso2.carbon.apimgt.migration.dto.ResourceScopeInfoDTO;
-import org.wso2.carbon.apimgt.migration.dto.ResourceScopeMappingDTO;
-import org.wso2.carbon.apimgt.migration.dto.UserRoleFromPermissionDTO;
+import org.wso2.carbon.apimgt.migration.dto.*;
```
Comment (msm1992) [inline migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/dao/APIMgtDAO.java:25]: here as well (* imports)

File: migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/client/MigrateFrom310.java (hunk 1 above)
Comment (CrowleyRajapakse) [reply migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/client/MigrateFrom310.java:28]: ack, will update

File: migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/dao/APIMgtDAO.java (hunk 2 above)
Comment (CrowleyRajapakse) [reply migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/dao/APIMgtDAO.java:25]: ack, will update

File: migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/client/MigrateFrom310.java (hunk 1 above)
Comment (CrowleyRajapakse) [reply migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/client/MigrateFrom310.java]: resolved

File: migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/dao/APIMgtDAO.java (hunk 2 above)
Comment (CrowleyRajapakse) [reply migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/dao/APIMgtDAO.java]: resolved

**Pull request merged**